openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -314,7 +314,7 @@ For security considerations, see (#credential-offer-security).
 This specification defines the following parameters for the Credential Offer object:
 
 * `credential_issuer`: REQUIRED. The URL of the Credential Issuer, as defined in (#credential-issuer-identifier), from which the Wallet is requested to obtain one or more Credentials. The Wallet uses it to obtain the Credential Issuer's Metadata following the steps defined in (#credential-issuer-wellknown).
-* `credentials`: REQUIRED. A JSON array, where every entry is a JSON object or a JSON string. If the entry is an object, the object contains the data related to a certain credential type the Wallet MAY request. Each object MUST contain a `format` Claim determining the format of the credential to be requested and further parameters characterizing the type of the credential to be requested as defined in (#format_profiles). If the entry is a string, the string value MUST be one of the `scope` values in one of the objects in the `credentials_supported` Credential Issuer metadata parameter. When processing, the Wallet MUST resolve this string value to the respective object.
+* `credentials`: REQUIRED. An array of strings that each identify one of the keys in the key/value pairs stored in the `credentials_supported` Credential Issuer metadata property. The Wallet uses this string value to obtain the respective object that contains information about the Credential being offered as defined in (#credential-issuer-parameters). For example, this string value can be used to obtain `scope` value to be used in the Authorization Request.
```

Comment:
```suggestion
* `credentials`: REQUIRED. An array of unique strings that each identify one of the keys in the key/value pairs stored in the `credentials_supported` Credential Issuer metadata property. The Wallet uses this string value to obtain the respective object that contains information about the Credential being offered as defined in (#credential-issuer-parameters). For example, this string value can be used to obtain `scope` value to be used in the Authorization Request.
```

Reply:
when saying unique, I think we need to be a little more specific..? I think we would want `unique within that issuer, but not globally unique` ?

Reply:
I suggest replacing `key` with `name` in when refering to members of JSON objects. I.e. `name/value pairs` instead of `key/value` pairs. Both https://www.json.org and https://www.rfc-editor.org/rfc/rfc8259.html seem to use the `name` terminology.

Reply:
@Sakurann Yeah, but that is more or less assumed by just adding unique. But always better to be very explicit about it

Reply:
```suggestion
* `credentials`: REQUIRED. An array of unique strings that each identify one of the keys in the name/value pairs stored in the `credentials_supported` Credential Issuer metadata property. The Wallet uses this string value to obtain the respective object that contains information about the Credential being offered as defined in (#credential-issuer-parameters). For example, this string value can be used to obtain `scope` value to be used in the Authorization Request.
```
agreed to change to `name/value pairs` and to keep `unique` (don't have a better text proposal to clarify)